submissions/sprint_a/app.py

```diff
@@ -0,0 +1,12 @@
+from sprint_a import app
+from flask import jsonify
+
+
+@app.route('/')
+def generate_epithets():
+    return jsonify({"epithets": []})
```

Comment:
Good use of Flask's components. jsonify is the preferred method for returning JSON responses.
http://flask.pocoo.org/docs/1.0/api/#flask.json.jsonify